install.sh

```diff
@@ -47,12 +47,14 @@ execute() {
   # hash_sha256_verify "${tmpdir}/${TARBALL}" "${tmpdir}/${CHECKSUM}"
   srcdir="${tmpdir}"
   (cd "${tmpdir}" && unzip -o "${TARBALL}")
-  install -d -m 775 "${BINDIR}" 2> /dev/null || install -d "${BINDIR}"
+  log_debug "setting up bindir: $BINDIR"
+  mkdir -p $BINDIR
```

Comment:
This `install` was the main problem.  We could use `sudo`, it'll work in GH, but I don't know if we want to do that for everyone else.

Reply:
This install makes sense that it's the issue. If the user's CI doesn't have a `usr/local/bin` is the whole script going to break? I don't know if that's a valid issue, but we should be aware of it before merging.

Reply:
The issue is that it already exists, and the `install` is trying to control it.  In CI, we could just `sudo install`, but I don't want to ship that as the installation script.  We just need the directory to exist, so we can simply use `mkdir -p`

The change to GH was that the directory used to not exist, but now it does.  Or ownership of the directory changed, it's hard to tell.

If the user doesn't have the directory, we will just make it.  We should probably fallback to `sudo mkdir`.

I'm going to refactor this script anyway, just so we can clear up the existing issue, then we can talk in detail about what we should be doing.